addon/services/best-language.ts

```diff
@@ -0,0 +1,121 @@
+import {computed} from '@ember-decorators/object';
+import {getOwner} from '@ember/application';
+import Service from '@ember/service';
+
+declare class FastBoot {
+  isFastBoot: boolean;
+  request: Request;
+}
+
+interface Language {
+  language: string;
+  baseLanguage: string;
+  score: number;
+}
+
+interface LanguageWithScore {
+  language: string;
```

Comment:
Ça sert à quoi `LanguageWithScore` si on a déjà le score dans `Language`?

Reply:
It’s the type we use before having `baseLanguage` in the object, I’m open to suggestions if you have a better name though!

Reply:
We could only have a `Language` interface with the `baseLanguage` optional.

Reply:
You’re right 🤦‍♂️ I need to step up my TS game! 😄